README.md

```diff
@@ -30,27 +26,27 @@ To get a local copy up and running follow these simple example steps.
 
 ### Install
 
+- To access this site, you can clone it to you local environment
+
 ### Usage
 
 ### Run tests
 
 ### Deployment
 
```

Comment:
- [ ] I would recommend you clean up your documentation. If these sections are not in use, then kindly remove them.

Reply:
Fixed